magento/pwa-studio
# [PWA-720] Error Case Tests

## Description

Test cases for #2495, merged ahead of tests to unblock regression.

## Related Issue





* [[PWA-720](https://jira.corp.magento.com/browse/PWA-720)] 

## Acceptance 



### Verification Stakeholders

### Specification



### Verification Steps


Since all files changes are all test files (`*.spec.js` + snapshots), this really just needs green builds to verify.
1. `yarn test`
2. Verify builds pass

## Screenshots / Screen Captures (if appropriate)

## Checklist

* I have updated the documentation accordingly, if necessary.
